Languages/English/Keyed/AllKeyed_Help.xml

```diff
@@ -46,23 +46,23 @@ Section 1.1 - Physical properties
-  <PRF_DeepQuarry_mkII_helpText>The Tier 2 Pulse Drill will automatically produce one random resource every three hours. This is based on the actual commonality of those materials in the world, so expect to see a lot of rock chunks relative to the amount of actually valuable stuff you get. The auger drill can anger insectoids and draw infestations.</PRF_DeepQuarry_mkII_helpText>
-  <PRF_DeepQuarry_mkIII_HelpText>The Tier 3 Pulse Drill Mark 2 will automatically produce random resources for you on a regular basis, but it will not produce rock chunks or cause infestations, meaning that the return in valuable materials is much higher.\n\nMods that add additional materials are automatically included, but there's no way to tell if a mod is adding more rock chunks. If this drill is producing rock chunks, the mod in question will require a patch.</PRF_DeepQuarry_mkIII_HelpText>
-  <PRF_BillTypeMiner_I_HelpText>The Tier 3 Plasma Drill will produce specific resources by demand. Select what you'd like using the Bills menu. The Plasma Drill will not draw insectoid infestations to you.</PRF_BillTypeMiner_I_HelpText>
+  <PRF_DeepQuarry_mkI_HelpText>The tier 1 auger drill will automatically produce one random resource every eight hours. This is based on the actual commonality of those materials in the world, so expect to see a lot of rock chunks relative to the amount of actually valuable stuff you get. The auger drill can anger insectoids and draw infestations.</PRF_DeepQuarry_mkI_HelpText>
+  <PRF_DeepQuarry_mkII_helpText>The tier 2 pulse drill Mk.I will automatically produce one random resource every three hours. This is based on the actual commonality of those materials in the world, so expect to see a lot of rock chunks relative to the amount of actually valuable stuff you get. The auger drill can anger insectoids and draw infestations.</PRF_DeepQuarry_mkII_helpText>
+  <PRF_DeepQuarry_mkIII_HelpText>The tier 3 pulse drill Mk.II will automatically produce random resources for you on a regular basis, but it will not produce rock chunks or cause infestations, meaning that the return in valuable materials is much higher.\n\nMods that add additional materials are automatically included, but there's no way to tell if a mod is adding more rock chunks. If this drill is producing rock chunks, the mod in question will require a patch.</PRF_DeepQuarry_mkIII_HelpText>
+  <PRF_BillTypeMiner_I_HelpText>The tier 3 plasma drill will produce specific resources by demand. Select what you'd like using the "Bills" tab. The plasma drill will not draw insectoid infestations to you.</PRF_BillTypeMiner_I_HelpText>

```

Comment:
I think `tier` should be Capitalized.

Not sure if we need to qute Bills

Reply:
The form `"Bills" tab` is used in other 9 places. Will double check "tier".

Reply:
Seems it is not mandatory to capitalize words like "tier". Also there are many examples of it in lowercase in mods like Alpha Bees and Vanilla Vehicles Expanded for example.

Reply:
Another example from rimworld: "Higher tier subcores can be created by building a subcore softscanner or a subcore ripscanner."